src/node.rs

```diff
@@ -176,11 +178,122 @@ pub trait AudioControllableSourceNode {
     }
 }
 
+const TABLE_LENGTH_USIZE: usize = 2048;
```

Comment:
Perhaps move all (private) oscillator internals in a separate mod? The node.rs file is huge already

Reply:
I put all related oscillator code into oscillator.rs